package.json

```diff
@@ -61,9 +61,10 @@
     "start": "npm run create-linux-dev-shortcut && electron dist/ --disable-dev-mode",
     "restart": "run-s build start",
     "watch": "npm run create-linux-dev-shortcut && node scripts/watch.js",
-    "package": "run-s package:windows package:windows-nsis package:mac-with-universal package:mas package:linux",
-    "package:windows": "npm run build-prod && electron-builder --win zip --x64 --ia32 --arm64 --publish=never",
-    "package:windows-nsis": "npm run build-prod-upgrade && electron-builder --win zip nsis --x64 --ia32 --arm64 --publish=never",
+    "package": "run-s package:windows package:mac-with-universal package:mas package:linux",
+    "package:windows": "run-s package:windows-*",
+    "package:windows-zip": "npm run build-prod && electron-builder --win zip --x64 --arm64 --publish=never",
+    "package:windows-installers": "npm run build-prod-upgrade && electron-builder --win nsis --x64 --arm64 --publish=never && node scripts/cp_msi_app_update.js && electron-builder --win msi --x64 --arm64 --publish=never",
```

Comment:
It is mandatory for the MSI installer that we build the NSIS first. This is so we can copy the `app-update.yml` file necessary to tell the app how to auto-update for the MSI build.